core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -295,4 +295,12 @@ package object config {
         "above this threshold. This is to avoid a giant request takes too much memory.")
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("200m")
+
+  private[spark] val TASK_METRICS_TRACK_UPDATED_BLOCK_STATUSES =
+    ConfigBuilder("spark.taskMetrics.trackUpdatedBlockStatuses")
```

Comment:
you can document it in `configuration.md`.

Actually can we turn it off by default? I think this is feature is useless for most of users. cc @JoshRosen